codecov_auth/views/logout.py

```diff
@@ -1,11 +1,21 @@
 from django.conf import settings
 from django.contrib.auth import logout
-from django.shortcuts import redirect
+from django.http import HttpRequest
+from django.shortcuts import HttpResponse, redirect
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
 
 
-def logout_view(request, **kwargs):
-    redirect_url = settings.CODECOV_DASHBOARD_URL
-    response = redirect(redirect_url)
+@api_view(["GET", "POST"])
+def logout_view(request: HttpRequest, **kwargs: str) -> HttpResponse:
+    print(kwargs)
+    if request.method == "POST":
+        response = Response(status=205)
```

Comment:
205 is interesting, never seen that but I like it

Reply:
Lol yep. Love using http statuses to their fullest 😄 

I wanted to do a 301 and redirect to login, but I was having issues with CORS because the request includes creds.

Reply:
ah yeah I wouldn't worry about that. We can also push redirects to the frontend if needed (which I think is current behavior?)

Reply:
Current behavior follows the redirect returned in the this file. With the POST will have to do:
1. POST to logout
2. redirect manually to /login

but I think this is just fine personally!